### ISSUE TITLE
Various styling improvements

### ISSUE THREAD

**Issue by brentguf** [WelcometoMyGarden/welcometomygarden#47]
Taking the minimal work approach. If you want to tackle an issue separately, please make a new issue and inform me to delete it here.

1. "Learn more" scroll on landing page scrolls page up too much
2. If you open an FAQ item on the landing page, there should be a border bottom underneat the content/answer
3. Green section grows together with FAQ item
4. More padding bottom on Rules/FAQ pages
5. More margin bottom underneath 'Your garden' paragraph (above button, welcometomygarden.org/account)
6. Faq list items were not made to be full page width, so the component padding needs to be adjusted for wider screens
7. Margin bottom underneath map when adding garden
8. Reset password copy: space should not be part of link
9. Use our green instead of orange for links as this can be hard to read (a11y, or just in general)

**Comment by MichielLeyman** [comment]
Thanks Brent!

I think of 3 more as a feature than a bug.

Not sure what you mean by 6.

**Comment by wardbeyens** [comment]
https://github.com/WelcometoMyGarden/welcometomygarden/pull/118

- [ ] 1. "Learn more" scroll on landing page scrolls page up too much
- [ ] 2. If you open an FAQ item on the landing page, there should be a border bottom underneat the content/answer
- [ ] 3. Green section grows together with FAQ item
- [ ] 4. More padding bottom on Rules/FAQ pages
- [ ] 5. More margin bottom underneath 'Your garden' paragraph (above button, welcometomygarden.org/account)
- [ ] 6. Faq list items were not made to be full page width, so the component padding needs to be adjusted for wider screens
- [ ] 7. Margin bottom underneath map when adding garden
- [ ] 8. Reset password copy: space should not be part of link
- [ ] 9. Use our green instead of orange for links as this can be hard to read (a11y, or just in general)

**Comment by MichielLeyman** [comment]
Closing this issue as most has been fixed by Ward in #118. It's unclear which of the other items remain, so please open a separate issue for each if necessary.
